snap/README.md

```diff
@@ -227,30 +226,19 @@ not addressable from another system). In order to make a service accessible remo
 
 ##### JWT tokens
 
-Before the API Gateway can be used, a user and group must be created, and a JWT access token generated. This can be accomplised via the
-`secrets-config` command, or by using `snap set` commands.
+Before the API Gateway can be used, a user and group must be created and a JWT access token generated.
 
-The first step is to add a user. You need to create a public/private keypair, which can be done with
+1. The first step is to create a public/private keypair for the new user, which can be done with
 
 ```bash
 # Create private key:
-$ openssl ecparam -genkey -name prime256v1 -noout -out private.pem
+openssl ecparam -genkey -name prime256v1 -noout -out private.pem
```

Comment:
openssl ecparam -genkey -name prime256v1 -noout -out private.pem
ecparam: Can't open "private.pem" for writing, Permission denied

(I think it need to be used with `sudo`)

Reply:
sudo should not be needed - but you need to run these commands in a working directory in your home directory that you have write access to... so you were probably running the command somewhere else?

Reply:
Thanks! I was in the checkbox directory, which already contains used key pairs. sudo should not be needed.